appengine/sendgrid/README.md

```diff
@@ -16,7 +18,8 @@ Before you can run or deploy the sample, you will need to do the following:
 1. [Create a SendGrid Account](http://sendgrid.com/partner/google). As of
 September 2015, Google users start with 25,000 free emails per month.
 1. Configure your SendGrid settings in the environment variables section in
-`app.yaml`.
+`app.standard.yaml` (if you are deploying to App Engine Standard Environment) or
```

Comment:
The inline code comment in the [/sendgrid/app.js file](https://cloud.google.com/appengine/docs/flexible/nodejs/sending-emails-with-sendgrid#example) refers to "app.yaml" which could be confusing now that we have these two environment specific files, which are named differently.

To assist with learning this concept of "uniquely-named, service-specific app.yaml files", perhaps add to the "app.js" file:
**(for example, app.standard.yaml or app.flexible.yaml)**

-> therefore you instead have:

*"....variables are set by app.yaml (for example, app.standard.yaml or app.flexible.yaml) when running on ......"*

Reply:
Done :)